api-reference/beta/api/alert_update.md

```diff
@@ -0,0 +1,163 @@
+# Update alert
+
+Update an editable **alert** property within any integrated solution to keep alert status and assignments in sync across solutions. This method updates any solution that has a record of the referenced alert ID. 
+
+>**Note:** You must include the alert ID as a parameter with this method.
+

```

Comment:
I changed this from bold text to a Note format, but to double-check, does this need to be called out? Can this be regular text, or is it clear enough from the HTTP request and examples that the ID needs to be included?